docs/content/configuration/creating_volumereplicationclass.md

```diff
@@ -24,6 +32,20 @@ spec:
     replication.storage.openshift.io/replication-secret-name: demo-secret
     replication.storage.openshift.io/replication-secret-namespace: default
 ```
+**Example 2: VolumeReplicationClass using Spectrum Virtualize policy-based replication**
+
+```
+apiVersion: replication.storage.openshift.io/v1alpha1
+kind: VolumeReplicationClass
```

Comment:
what about VolumeReplication YAML?

Reply:
@ELENAGER  does anything need to be added for EAR?

Reply:
Nothing needs to be added to volume replication yaml file. "replicationHandle" must NOT be there, otherwise user will receive an error

Reply:
thanks, @ELENAGER 
@annaparienti  can we close this thread?

Reply:
@ELENAGER, what about the new dataSource for VG instead of PVC?

Reply:
@annaparienti it is still not implemented